docs/how_to/use_estimator.rst

```diff
@@ -0,0 +1,140 @@
+###########################
+Use the Estimator primitive
+###########################
+
+This guide shows how to get the expected value of an observable for a given quantum circuit with the :class:`~qiskit.primitives.Estimator` primitive.
+
+
+Initialize observable
+=====================
+
+The first step is to define the observable whose expected value you want to compute. This observable can be any ``BaseOperator``, like the operators from :mod:`qiskit.quantum_info`.
+Among them it is preferable to use :class:`~qiskit.quantum_info.SparsePauliOp`.
+
+.. testcode::
+
+    from qiskit.quantum_info import SparsePauliOp
+
+    obs = SparsePauliOp(["II", "XX", "YY", "ZZ"], [1, 1, -1, 1])
+
+Initialize quantum circuits
```

Comment:
```suggestion
Initialize the quantum circuit
```
This section only mentions a single circuit later on so i think this should be singular?